components/application-mgt/org.wso2.carbon.identity.application.mgt/src/main/java/org/wso2/carbon/identity/application/mgt/ApplicationResourceManager.java

```diff
@@ -70,6 +85,14 @@ ServiceProvider getApplicationByResourceId(String resourceId,
     void updateApplicationByResourceId(String resourceId, ServiceProvider updatedApplication,
                                        String tenantDomain, String username)
             throws IdentityApplicationManagementException;
+    
+    default void updateApplicationByResourceId(String resourceId, ServiceProvider application,
```

Comment:
Add the method description

Reply:
Fixed with https://github.com/wso2/carbon-identity-framework/pull/5146/commits/52ad1c5d357df775755cc1d1f76c7a76c4dd50ae